app_test.py

```diff
@@ -0,0 +1,22 @@
+from app import app
+
+from flask_testing import TestCase
+
+
+class siteTest(TestCase):
+
+    def create_app(self):
+        app.config['TESTING'] = True
+        app.config['LIVESERVER_TIMEOUT'] = 10
+        app.allow_subdomain_redirects = True
+        return app
+
+
+class RootEndpoint(siteTest):
```

Comment:
While PEP8 doesn't really specify whether or not to put a blank line after the class definition line, we must at least be consistent about whether we do or not.